lms/djangoapps/verify_student/models.py

```diff
@@ -1384,6 +1399,19 @@ class IcrvStatusEmailsConfiguration(ConfigurationModel):
     pass
 
 
+class StudentVerificationConfiguration(ConfigurationModel):
+    """Student Verification Configuration"""
+    days_good_for = models.PositiveIntegerField(
+        default=365, help_text="How many days is a verification good for"
+    )
+    expiring_soon_window_in_days = models.PositiveIntegerField(
+        default=28, help_text="The window within which a verification is considered to be expiring soon"
+    )
+
+    def __unicode__(self):
+        return "Student Verification Configuration"
```

Comment:
A useful string representation of the model will include values from the model's fields.